lib/util/schema-utils.ts

```diff
@@ -258,7 +258,7 @@ export const MultidocYaml = z.string().transform((str, ctx): JsonArray => {
 
 export const Toml = z.string().transform((str, ctx) => {
   try {
-    return parseToml(str);
+    return parseToml(str, { removeTemplates: true });
```

Comment:
I don't like to do this by default. @zharinov how to best handle such options 🤔 

Maybe we should export a `toml` function which allows to pass parder options? would this help for yaml on gitlab too?

Reply:
I think it isn't necessary to couple with TOML parsing.

Let's use `stripTemplates(parseToml(...))` or `StripTemplates.pipe(Toml).pipe(z.object(...))` instead.

Reply:
Yes, these transforms could lead to errors that are hard to detect.
Let's keep it localized to the Python ecosystem:
- https://github.com/renovatebot/renovate/pull/34447